lib/private/Session/CryptoWrapper.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
 					[
 						'expires' => 0,
 						'path' => $webRoot,
-						'domain' => '',
+						'domain' => $this->config->getSystemValueString('cookie_domain'),
```

Comment:
As you can see in psalm output, there is no config property in this class.

So you need to either inject it in the constructor, or directly pull it here:
```suggestion
						'domain' => \OCP\Server::get(\OCP\IConfig::class)->getSystemValueString('cookie_domain'),
```

Reply:
OK! Included.